uai2017/process/uaidata.js

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
I like the idea!

What is this a js module with variables or a json object? I don't understand what it is. 

and why are they in all CAPS?